src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/OrderingPartyController.cs

```diff
@@ -55,25 +55,23 @@ public async Task<ActionResult> UpdateAsync(CallOffId callOffId, OrderingPartyMo
             if (model is null)
                 throw new ArgumentNullException(nameof(model));
 
-            var order = await context.Order
-                .Where(o => o.Id == callOffId.Id)
-                .Include(o => o.OrderingParty).ThenInclude(p => p.Address)
-                .Include(o => o.OrderingPartyContact)
-                .SingleOrDefaultAsync();
+            var order = await orderingPartyService.GetOrder(callOffId);
 
             if (order is null)
                 return NotFound();
 
-            var orderingParty = order.OrderingParty;
-            orderingParty.Name = model.Name;
-            orderingParty.OdsCode = model.OdsCode;
+            OrderingParty orderingParty = new OrderingParty
+                {
+                    Name = model.Name,
+                    OdsCode = model.OdsCode,
+                    Address = contactDetailsService.AddOrUpdateAddress(order.OrderingParty.Address, model.Address),
+                };
```

Comment:
We can use `var` here. Also, can you reduce the indent of the braces by one tab? (Just to keep the styling consistent with other places in the code.)

Reply:
Updated with `var` keyword and updated indentation